<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClasaPlay</class>
 <widget class="QMainWindow" name="ClasaPlay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Play</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color:rgb(255, 170, 0)
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Poze/Resources/blank.jpg);</string>
   </property>
   <widget class="QLabel" name="Question">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>781</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 italic 9pt &quot;Roman&quot;;
border-image: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="AnswerA">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>220</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: transparent;

}</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AnswerB">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>300</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: transparent;</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AnswerC">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>380</y>
      <width>251</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: transparent;</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AnswerD">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>460</y>
      <width>251</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: transparent;</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>461</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: transparent;</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="AvantajJumatate">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>360</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: transparent;</string>
    </property>
    <property name="text">
     <string>50/50</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Registration.qrc"/>
 </resources>
 <connections/>
</ui>
